### PR TITLE
Fix config

### PR DESCRIPTION
**Pull request by pmlopes**
The SSO url was pointing to the spring server:

After the change the current code works:

```
[plopes@jetdrone quick_secured_rest-vertx]$ ./scripts/curl/token_req.sh https://secure-sso-ssovertx.e8ca.engint.openshiftapps.com http://localhost:8080
>>> Greeting
{"id":9,"content":"Hello, World!"}>>> Greeting Customized Message
{"id":10,"content":"Hello, Spring!"}[plopes@jetdrone quick_secured_rest-vertx]$ ./scripts/curl/token_req.sh https://secure-sso-ssovertx.e8ca.engint.openshiftapps.
com http://localhost:8080
>>> Greeting
{"id":11,"content":"Hello, World!"}>>> Greeting Customized Message
{"id":12,"content":"Hello, Spring!"}[plopes@jetdrone quick_secured_rest-vertx]$
[plopes@jetdrone quick_secured_rest-vertx]$ ./scripts/curl/token_req.sh https://secure-sso-ssovertx.e8ca.engint.openshiftapps.com http://localhost:8080
>>> Greeting
{"id":13,"content":"Hello, World!"}>>> Greeting Customized Message
{"id":14,"content":"Hello, Spring!"}[plopes@jetdrone quick_secured_rest-vertx]$
```

### CODE REVIEW

**Comment by cmoulliard** [comment]
@pmlopes The README file is a copy/paste of the SpringBoot Secured project & will be completely reviewed for Obsidian Alpha2

**Comment by pmlopes** [comment]
True but the other 5 files are important to point to the correct server and those are the test scripts + fabric8 deployment configs

File: README.md [hunk 1]
```diff
@@ -103,8 +103,8 @@ Depending which tool you prefer to use (curl or httpie), use one of bash files a
 the address of the Red Hat Secured SSO Server and the Secured Vert.x Application.
 
 ```
-./scripts/httpie/token_req.sh https://secure-sso-sso.e8ca.engint.openshiftapps.com http://vertx-rest-sso.e8ca.engint.openshiftapps.com
-./scripts/curl/token_req.sh https://secure-sso-sso.e8ca.engint.openshiftapps.com http://vertx-rest-sso.e8ca.engint.openshiftapps.com
+./scripts/httpie/token_req.sh https://secure-sso-ssovertx.e8ca.engint.openshiftapps.com http://vertx-rest-sso.e8ca.engint.openshiftapps.com
+./scripts/curl/token_req.sh https://secure-sso-ssovertx.e8ca.engint.openshiftapps.com http://localhost:8080
```
Comment (edewit) [inline README.md]: shouldn't this be `http://vertx-rest-sso.e8ca.engint.openshiftapps.com` instead of `http://localhost:8080`

Comment (cmoulliard) [inline README.md]: yes. localhost should be replaced by the name of the server running in OSO

**Pull request merged**